internal/monitors/kubernetes/kubernetes.go

```diff
@@ -5,5 +5,6 @@ import (
 	_ "github.com/signalfx/signalfx-agent/internal/monitors/kubernetes/apiserver"
```

Comment:
Is it possible to rename this file to modules.go?

Reply:
Yeah I always have a hard time finding this file.